src-electron/db/zap-schema.sql

```diff
@@ -596,6 +596,9 @@ CREATE TABLE IF NOT EXISTS "ATOMIC" (
   "IS_LONG" integer default false,
   "IS_CHAR" integer default false,
   "IS_SIGNED" integer default false,
+  "IS_COMPOSITE" integer default false,
+  "IS_FLOAT" integer default false,
+  "BASE_TYPE" text,
```

Comment:
This allows BASE_TYPE to be null?  Or if not, how is "no base type" represented?  Empty string?

Reply:
It will be null in this case.